src/ktn_dodger.erl

```diff
@@ -394,9 +400,9 @@ parse_form(Parser, Ts, L1, NoFail, Opt) ->
             IoErr = io_error(L1, Term),
             {error, IoErr, L1};
         {parse_error, _IoErr} when NoFail ->
-            {ok,
-             erl_syntax:set_pos(
-                 erl_syntax:text(tokens_to_string(Ts)), start_pos(Ts, L1)),
+            {ok, erl_syntax:set_pos(
+                    erl_syntax:text(tokens_to_string(Ts)),
+                    erl_anno:new(start_pos(Ts, L1))),
```

Comment:
Indentation seems off.

Reply:
Yeah… because this module is ignored by the formatter. I'll fix as much as I can _by hand_.